### ISSUE TITLE
Primal Beast Immortal effect can not be turned on without equipping the item

### ISSUE THREAD

**Issue by ekoveko** [ValveSoftware/Dota2-Gameplay#5595]
### Cosmetic Item Name

Age of Attrition - Head

### Description

You can not use the custom spell that comes with the cosmetic without equipping it, as opposed to all other immortal items

### Example Match ID

-

### Screenshots

![image](https://user-images.githubusercontent.com/118853459/203414874-f33871d9-ee02-427e-9e90-720e668e1456.png)

**Comment by DefStrOke666** [comment]
Just got here to write the same bug report

**Comment by Jessicaalban86** [comment]
Nope, it's a bug. PB is likely an oversight as there were no non-immortal items until now. Tiny is bugged too, as you can currently only equip the prestige immortal effect for 1 of his 4 stages.
